libraries/rib-base/src/test/java/com/badoo/ribs/core/lifecycle/CappedLifecycleTest.kt

```diff
@@ -69,6 +69,7 @@ class CappedLifecycleTest {
     @Test
     fun `when external is INITIALIZED, internal is after onDestroy(), effective is DESTROYED`() {
         setExternal(INITIALIZED)
+        cappedLifecycle.onCreate()
```

Comment:
Same as above. You cannot move from INITIALISED to DESTROYED. Perhaps some of these tests should be removed as they are not valid?